boards/xtensa/qemu_xtensa/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Kconfig - Xtensa Qemu board configuration
+#
+# Copyright (c) 2016 Cadence Design Systems, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config XTENSA_XTSC_INC
+	string "Xtensa XTSC extension include file"
+	depends on SIMULATOR_XTENSA
+	help
+	If you want to use xtsc-run with an extended core, then you can put here the
+	path to the include file of the extension model. This path can either be
+	absolute or relative to the execution directory.
```

Comment:
I don't think that QEMU supports SystemC extension, so this is probably to be removed.
However please check with Max first.